src/test/java/guitests/MenuControlTest.java

```diff
@@ -117,6 +117,9 @@ public void menuControlTest() {
 
         // Testing First Panel selected
         assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT1).release(KeyCode.DIGIT1).release(KeyCode.CONTROL);
+        assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        // TODO (currently not possible) test that panelsScrollPane.getHValue() is 0
```

Comment:
@dariusf I added this as todo because when I implemented the function, the panel that is focused was not in view, i.e. the pane doesn't scroll to the one in focus.
However, there is no current functionality to check the caret/pane position right now unless I make a public method to do so, which breaks encapsulation. Is the comment sufficient?